### DISCUSSION TITLE
部分UP主视频无法下载

### DISCUSSION THREAD

**Discussion by YanChiCu** [yutto-dev/yutto#192] [General]
大部分UP主空间视频可正常下载，部分UP主视频空间的下载进度条显示正常，合并结束后只剩下了ass和srt
重登陆更换cookie和更换清晰度无效
无法下载的UP主视频空间:
https://space.bilibili.com/352632088/video
https://space.bilibili.com/585267/video

**Comment by SigureMo** [comment]
无法复现

<img width="783" alt="image" src="https://github.com/yutto-dev/yutto/assets/38436475/c479bbef-40bd-4159-add5-25a2f9c6e0df">
